certbot/plugins/webroot_test.py

```diff
@@ -34,7 +34,9 @@ class AuthenticatorTest(unittest.TestCase):
 
     def setUp(self):
         from certbot.plugins.webroot import Authenticator
-        self.path = tempfile.mkdtemp()
+        self.workspace = tempfile.mkdtemp()
```

Comment:
Its permissions are unspecified. It depends of what are the actual ACEs for the current general temp dir, since the directory created by mkdtemp will inherit them. Instead I prefer to create my own directory in it, with a real control over the ACEs permissions created on it.

Reply:
Makes sense! Can you add an inline comment explaining this?

Reply:
Done.